osu.Game.Tests/Visual/Settings/TestSceneKeyBindingPanel.cs

```diff
@@ -71,7 +72,7 @@ public void TestClearButtonOnBindings()
 
             AddStep("click first row with two bindings", () =>
             {
-                multiBindingRow = panel.ChildrenOfType<KeyBindingRow>().First(row => row.Defaults.Count() > 1);
+                multiBindingRow = panel.ChildrenOfType<SettingsKeyBindingRow>().First(row => row.KeyBindingRow.Defaults.Count() > 1).KeyBindingRow;
```

Comment:
Not sure what these changes are about. `ChildrenOfType()` is recursive, so it should continue to work the same way it did even with the component changes, I think.